blues/app-accelerators
# feat: nf10 code simplification

# Problem Context

This web app has very similar functionality to the valve monitor web app minus the valve control and indicator of the valve's current state. Rewrite this app to reflect those changes and remove all valve indications from the code.

## Changes

* Rename all `valve` related references to instead be `flowRate` related.
* Remove all code that had to do with valve indications and controls from the app.
* Update the CSS to account for these changes and match the mockups.

## Screenshot (if applicable)

https://user-images.githubusercontent.com/20400845/208136291-672e6845-e113-4654-abb3-c3690e996603.mov

## Testing

Unit / integration / e2e tests?

No new automated tests added. All unit tests pass and code compiles.

Steps to test manually?

1. Fire up the PrismaDB 
2. Fire up the app and go to http://localhost:4000
3. See the app and make sure all fleet and device level env vars show up, can be changed, update, etc., etc.

Any other sorts of testing notes to include?

I also made some small updates to the CSS for the valve control app after I finished the CSS for this app (it's more accurate to the mockups now).

## Any other related PRs

n/a

## Ticket(s)

https://trello.com/c/OTdN93d1/144-nf10-remove-the-valve-control-code-from-web-app
